README.md

```diff
@@ -146,44 +63,45 @@ from appium import webdriver
 
 From there much of your test code will work with no change.
 
-As a base for the following code examples, the following sets up the [UnitTest](https://docs.python.org/2/library/unittest.html)
+As a base for the following code examples, the following sets up the [UnitTest](https://docs.python.org/3/library/unittest.html)
```

Comment:
(nits) I think `unittest` description is unnecessary as sample codes anymore
since functional tests in this repo was moved to `pytest` from `unittest` . (
https://github.com/appium/python-client/pull/488)